recipe/meta.yaml

```
@@ -8,12 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d7366c2e2293314112dcf2432108428a67b927b00005619feefc310d12d833f3
+  sha256: e321e263b6a97aafc0790c3cfb3c04656b7066e6738c37fffcca95d803c9fba5
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  skip: True  # [py<37]
+  skip: True  # [py<39]
```

```suggestion
  skip: True  # [py<38]
```
3.0.0 supports Python 3.8 and up. See https://github.com/dask/zict/blob/3.0.0/setup.cfg#L33 and https://inspector.pypi.io/project/zict/3.0.0/packages/80/ab/11a76c1e2126084fde2639514f24e6111b789b0bfa4fc6264a8975c7e1f1/zict-3.0.0-py2.py3-none-any.whl/zict-3.0.0.dist-info/METADATA.

---

got it! changed